exporter/terra/experiment/client.py

```diff
@@ -54,8 +42,8 @@ def write_metadata(self, metadata: MetadataResource, project_uuid: str):
         self.write_to_staging_bucket(dest_object_key, data_stream)
 
         # TODO2: patch dcpVersion        
-        #patch_url = metadata.metadata_json['_links']['self']['href']
-        #self.ingest_client.patch(patch_url, {"dcpVersion": metadata.dcp_version})
+        # patch_url = metadata.metadata_json['_links']['self']['href']
```

Comment:
Do we still need this?

Reply:
I would keep this until we review the metadata exporting functionality in the future.